src/app/employer/(home)/_components/Tablel.tsx

```diff
@@ -34,9 +34,15 @@ const Table: React.FC<Props> = ({ className }) => {
 
     const handleDelete = async (id: string) => {
         try {
-            await axios.delete(`https://carrerhub-backend.vercel.app/api/v1/job/${id}`);
-            setJobs(jobs.filter(job => job._id !== id));
-            toast.success("Job deleted successfully");
+            const response = await axios.delete(`https://carrerhub-backend.vercel.app/api/v1/job/${id}`, {
```

Comment:
donot harcode the url, extract it to a promise and use it with tanstack/react-query